packages/cms/pages/assests/asset-manager/asset-manager.tsx

```diff
@@ -650,20 +711,55 @@ export const AssetManager: FunctionComponent = () => {
                   )
 
                   return (
-                    <EuiFlexItem>
-                      <EuiCard
-                        onClick={() => {
-                          setIsFlyoutVisible(true)
-                          setActive(asset)
-                        }}
-                        textAlign="left"
-                        hasBorder
-                        image={asset.path}
-                        title={asset.file}
-                        description=""
-                        footer={cardFooterContent}
-                      />
-                    </EuiFlexItem>
+                    <AssetCardWrapper
+                      onMouseLeave={() => {
+                        setAssets(
```

Comment:
I think rather than mapping through all assets like this, we could have a single state to track this. In the asset card wrapper, have a `[hovered, setHovered]` state and handle this in there. The loop might cause jag on some devices.

Reply:
This is how I initially wanted to approach it, but  I can't use the hooks in a loop, and if I put it outside the loop, on hover, the icon shows on all the assets when I want it on that one asset. :sweat: